vars/xwikiBuild.groovy

```diff
@@ -335,11 +335,11 @@ private def computeMavenGoals(config)
 {
     def goals = config.goals
     if (!goals) {
-        // Use "deploy" goal for "master" and "stable-*" branches only and "install" for the rest.
+        // Use "deploy" goal for "master", "main" and "stable-*" branches only and "install" for the rest.
```

Comment:
Sounds good but we need a comment to explain why we support both "master" and "main", i.e. that GitHub has changed its default branch name from "master" to "main" and that we need to support both old repos (created with "master") and new ones (created with "main").